BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -151,6 +330,236 @@ public static string GetVersion(string file, byte[] fileContent, List<int> posit
             return $"{version}.{subVersion:00}.{subsubVersion:000}";
         }
 
+        public static string Get00000001TMPVersion(string firstMatchedString, IEnumerable<string> files)
+        {
+            if (firstMatchedString == null || !File.Exists(firstMatchedString))
+                return string.Empty;
+
+            // This file is present in most, if not all, SafeDisc protected discs. It seems to have very consistent file sizes, only being found to use three different file sizes in it's entire run.
+            FileInfo fi = new FileInfo(firstMatchedString);
+            switch (fi.Length)
+            {
+                // Found in Redump entries 37832 and 66005. 
+                case 20:
+                    return "1.00.025-1.41.001";
+                // Found in Redump entries 30555 and 58573.
+                case 2_048:
+                    return "1.45.011+ (CD) / Cactus Data Shield 300";
```

Comment:
Gonna admit that this combination scares me a little. So CDS _also_ uses the same tempfile and it has the same size? If you're going to do this, either put the check in the CDS check and/or remove the mention of CDS from here in the return. That will get too confusing too quickly.

Reply:
It is the same file name with the same size, from the same company as well. Will split the checks next PR.

Reply:
I'd strongly prefer changing that as a part of this PR since this version is likely to be considered "stable".

Reply:
Sorry, meant commit and not PR 😜